test/tests/kernels/diffusion.i

```diff
@@ -2,6 +2,8 @@
   type = MFEMMesh
   file = gold/mug.e
   dim = 3
+  serial_refine = 0
+  parallel_refine = 0
```

Comment:
Could these lines be deleted, since they're setting the default levels anyway?

Reply:
Yup, done!